dataland-frontend/src/components/pages/DataPrivacy.vue

```diff
@@ -1,38 +1,1534 @@
+        <span style="font-size: 10pt; line-height: 133%"
+          >Sie haben darüber hinaus das Recht gemäß Art. 22 DS-GVO keiner vollautomatisierten Entscheidungsfindung zu
+          unterliegen. Zur Begründung, Durchführung und Beendigung der Geschäftsbeziehung nutzen wir grundsätzlich keine
+          vollautomatisierte Entscheidungsfindung. Sollten wir diese Verfahren in Einzelfällen einsetzen (z.B. zur
+          Verbesserung unserer Produkte und Services), werden wir Sie hierüber und über Ihre diesbezüglichen Rechte
+          gesondert informieren, sofern dies gesetzlich vorgegeben ist.</span
+        >
+      </p>
+
+      <h1
+        style="
+          margin-top: 0cm;
+          margin-right: 0cm;
+          margin-bottom: 12pt;
+          margin-left: 1cm;
+          text-align: justify;
+          text-indent: -1cm;
+          line-height: 133%;
+        "
+      >
+        8.<span style="font: 7pt &quot;Times New Roman&quot;"
+          >&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span
+        >PFLICHT ZUR BEREITSTELLUNG VON DATEN
+      </h1>
+
+      <p class="MsoNormal" style="margin-bottom: 12pt; line-height: 133%">
+        <span style="font-size: 10pt; line-height: 133%"
+          >Im Rahmen unseres Onlineangebots sind wir auf die Verarbeitung solcher Nutzungsdaten angewiesen, die für die
+          Durchführung und Beendigung des Dienstes und zur Erfüllung der damit verbundenen Pflichten erforderlich sind.
+          Ohne die Erhebung der Nutzungsdaten sind wir und unsere Dienstleister nicht in der Lage, Ihnen unser
+          Online-Angebot zur Verfügung zu stellen. Die Bereitstellung personenbezogener Daten ist aber weder gesetzlich
+          noch vertraglich vorgeschrieben.</span
+        >
+      </p>
+
+      <h1
+        style="
+          margin-top: 0cm;
+          margin-right: 0cm;
+          margin-bottom: 12pt;
+          margin-left: 1cm;
+          text-align: justify;
+          text-indent: -1cm;
+          line-height: 133%;
+        "
+      >
+        9.<span style="font: 7pt &quot;Times New Roman&quot;"
+          >&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span
+        >PROFILING
+      </h1>
+
+      <p class="MsoNormal" style="margin-bottom: 12pt; line-height: 133%">
+        <span style="font-size: 10pt; line-height: 133%"
+          >Wir verarbeiten Ihre personenbezogenen Daten nicht automatisiert in einer Weise, dass sie Ihnen gegenüber
+          rechtliche Wirkung entfaltet oder sie in ähnlicher Weise erheblich beeinträchtigt.</span
+        >
+      </p>
+
+      <h1
+        style="
+          margin-top: 0cm;
+          margin-right: 0cm;
+          margin-bottom: 12pt;
+          margin-left: 1cm;
+          text-align: justify;
+          text-indent: -1cm;
+          line-height: 133%;
+        "
+      >
+        10.<span style="font: 7pt &quot;Times New Roman&quot;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>AKTUALITÄT
```

Comment:
Formatting especially ugly under mobile (Basically, all long Headers are ugly because of massive spaces filled in behind the heading number)

Reply:
I could just remove these uneccessary spaces. Or do you find the bring much benefit?

Reply:
Yeah just remove the non-breaking spaces.

Reply:
I left 1 everywhere